src/JSON.js

```diff
@@ -0,0 +1 @@
{"start":{"x":274,"y":155},"end":{"x":274,"y":155}},{"start":{"x":274,"y":155},"end":{"x":275,"y":155}},{"start":{"x":275,"y":155},"end":{"x":275,"y":155}},{"start":{"x":275,"y":155},"end":{"x":275,"y":155}},{"start":{"x":275,"y":155},"end":{"x":275,"y":155}},{"start":{"x":275,"y":155},"end":{"x":275,"y":155}},{"start":{"x":275,"y":155},"end":{"x":276,"y":155}},{"start":{"x":276,"y":155},"end":{"x":276,"y":155}},{"start":{"x":276,"y":155},"end":{"x":276,"y":155}},{"start":{"x":276,"y":155},"end":{"x":276,"y":155}},{"start":{"x":276,"y":155},"end":{"x":276,"y":155}},{"start":{"x":276,"y":155},"end":{"x":276,"y":155}},{"start":{"x":276,"y":155},"end":{"x":277,"y":155}},{"start":{"x":277,"y":155},"end":{"x":277,"y":155}},{"start":{"x":277,"y":155},"end":{"x":277,"y":155}},{"start":{"x":277,"y":155},"end":{"x":277,"y":155}},{"start":{"x":277,"y":155},"end":{"x":278,"y":155}},{"start":{"x":278,"y":155},"end":{"x":278,"y":155}},{"start":{"x":278,"y":155},"end":{"x":278,"y":155}},{"start":{"x":278,"y":155},"end":{"x":278,"y":155}},{"start":{"x":278,"y":155},"end":{"x":279,"y":155}},{"start":{"x":279,"y":155},"end":{"x":279,"y":155}},{"start":{"x":279,"y":155},"end":{"x":279,"y":155}},{"start":{"x":279,"y":155},"end":{"x":279,"y":155}},{"start":{"x":279,"y":155},"end":{"x":279,"y":155}},{"start":{"x":279,"y":155},"end":{"x":280,"y":155}},{"start":{"x":280,"y":155},"end":{"x":280,"y":155}},{"start":{"x":280,"y":155},"end":{"x":280,"y":155}},{"start":{"x":280,"y":155},"end":{"x":281,"y":155}},{"start":{"x":281,"y":155},"end":{"x":281,"y":155}},{"start":{"x":281,"y":155},"end":{"x":281,"y":155}},{"start":{"x":281,"y":155},"end":{"x":281,"y":155}},{"start":{"x":281,"y":155},"end":{"x":281,"y":155}},{"start":{"x":281,"y":155},"end":{"x":281,"y":155}},{"start":{"x":281,"y":155},"end":{"x":282,"y":155}},{"start":{"x":282,"y":155},"end":{"x":282,"y":155}},{"start":{"x":282,"y":155},"end":{"x":282,"y":155}},{"start":{"x":282,"y":155},"end":{"x":282,"y":155}},{"start":{"x":282,"y":155},"end":{"x":283,"y":155}},{"start":{"x":283,"y":155},"end":{"x":283,"y":155}},{"start":{"x":283,"y":155},"end":{"x":283,"y":155}},{"start":{"x":283,"y":155},"end":{"x":283,"y":155}},{"start":{"x":283,"y":155},"end":{"x":284,"y":155}},{"start":{"x":284,"y":155},"end":{"x":284,"y":155}},{"start":{"x":284,"y":155},"end":{"x":284,"y":155}},{"start":{"x":284,"y":155},"end":{"x":284,"y":155}},{"start":{"x":284,"y":155},"end":{"x":284,"y":155}},{"start":{"x":284,"y":155},"end":{"x":285,"y":155}},{"start":{"x":285,"y":155},"end":{"x":285,"y":155}},{"start":{"x":285,"y":155},"end":{"x":285,"y":155}},{"start":{"x":285,"y":155},"end":{"x":285,"y":156}},{"start":{"x":285,"y":156},"end":{"x":285,"y":156}},{"start":{"x":285,"y":156},"end":{"x":286,"y":156}},{"start":{"x":286,"y":156},"end":{"x":286,"y":156}},{"start":{"x":286,"y":156},"end":{"x":286,"y":156}},{"start":{"x":286,"y":156},"end":{"x":286,"y":156}},{"start":{"x":286,"y":156},"end":{"x":286,"y":156}},{"start":{"x":286,"y":156},"end":{"x
```

Comment:
you can save a `someData.json` file and then import it directly (rather than having `.js` file with a default export of some JSON)

Reply:
I couldn't figure out how to import it that way, stack overflow told me to do this

Reply:
Oh okay. Well if it works it works